audmetric/core/api.py

```diff
@@ -1186,28 +1186,73 @@ def weighted_confusion_error(
 def word_error_rate(
     truth: Sequence[Sequence[str]],
     prediction: Sequence[Sequence[str]],
+    *,
+    norm: str = "truth",
 ) -> float:
     r"""Word error rate based on edit distance.
 
+    The word error rate is computed
+    by aggregating the normalized edit distances
+    of each (truth, prediction)-pair
+    and averaging the aggregated score
+    by the number of pairs.
+
+    The normalized edit distance
+    of each (truth, prediction)-pair is computed
+    as the edit distance divided by a normalization factor n.
+    This represents the average editing cost per sequence item.
+
+    The value of n depends on the ``norm`` parameter.
+    If ``norm`` is ``"truth"``,
```

Comment:
```suggestion
    This represents the average editing cost per sequence item.
    The value of n depends on the ``norm`` parameter.
    
    If ``norm`` is ``"truth"``,
```

Then it will look like this:

![image](https://github.com/user-attachments/assets/3b1f00d7-154f-4b4c-b2ae-7e2eb4ef8f0f)

Reply:
This is better. Our edits have crossed and I did a similar thing. 
Like this I am also happy as the explanations of "truth" and "longest" is easy to spot and understand